dex/src/main/scala/com/wavesplatform/dex/model/OrderInfo.scala

```diff
@@ -498,7 +508,7 @@ object OrderInfo {
       assetPair = AssetPair(buf.getAssetId, buf.getAssetId),
       orderType = buf.getAcceptedOrderType,
       orderVersion = buf.get(),
-      totalExecutedPriceAssets = buf.getLong()
+      avgWeighedPriceNominator = new BigInteger(buf.getBytes)
```

Comment:
This should not work. And probably tests fail because of this. `buf.getBytes` contains a length of array and the array itself.

Reply:
No, everything will [work correctly](https://github.com/wavesplatform/matcher/blob/master/dex/src/main/scala/com/wavesplatform/dex/codecs/ByteBufferCodecs.scala#L14)